zerver/lib/events.py

```diff
@@ -321,6 +322,10 @@ def fetch_initial_state_data(
         state["max_stream_description_length"] = Stream.MAX_DESCRIPTION_LENGTH
         state["max_topic_length"] = MAX_TOPIC_NAME_LENGTH
         state["max_message_length"] = settings.MAX_MESSAGE_LENGTH
+        if realm.demo_organization_scheduled_deletion_date is not None:
+            state["demo_organization_scheduled_deletion_date"] = datetime_to_timestamp(
+                realm.demo_organization_scheduled_deletion_date
+            )
```

Comment:
@gnprice do you think API clients would prefer for this field to be always present (and just `null` for non-demo organizations)?

Reply:
I'm now thinking the current design is probably fine; I'll go ahead and merge as written, since it should be unproblematic to change as it'll probably be a few weeks before these can exist anyway.